src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -475,6 +477,14 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </Page>
+    <None Include="UI\Themes\Modern\InPorts.xaml">
+      <SubType>Designer</SubType>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
This doesn't seem to be true for the other xaml files referenced in this project then why enable it for these?

Reply:
The items we load programmatically with the SharedResourceDictionary are copied to output